exposed-version-catalog/build.gradle.kts

```diff
@@ -0,0 +1,95 @@
+plugins {
+    `version-catalog`
+    `maven-publish`
+    signing
+}
+
+group = "org.jetbrains.exposed"
+version = rootProject.version
+
+catalog {
+    versionCatalog {
+        // Core modules
+        library("core", "org.jetbrains.exposed:exposed-core:${version}")
+        library("jdbc", "org.jetbrains.exposed:exposed-jdbc:${version}")
+        library("dao", "org.jetbrains.exposed:exposed-dao:${version}")
+        
+        // Date/time modules
+        library("java-time", "org.jetbrains.exposed:exposed-java-time:${version}")
+        library("joda-time", "org.jetbrains.exposed:exposed-jodatime:${version}")
+        library("kotlin.datetime", "org.jetbrains.exposed:exposed-kotlin-datetime:${version}")
+        
+        // Additional modules
+        library("json", "org.jetbrains.exposed:exposed-json:${version}")
+        library("money", "org.jetbrains.exposed:exposed-money:${version}")
+        library("crypt", "org.jetbrains.exposed:exposed-crypt:${version}")
+        library("migration", "org.jetbrains.exposed:exposed-migration:${version}")
+        
+        // Spring integration
+        library("spring", "org.jetbrains.exposed:spring-transaction:${version}")
+        library("spring.boot.starter", "org.jetbrains.exposed:exposed-spring-boot-starter:${version}")
+    }
+}
+
+publishing {
```

Comment:
https://github.com/jetbrains-junie publishing needs to be setup in buildSrc, in Publishing.kt. Check the existing usage, and update this configuration.